From 7b7879924719620f3d77eed0de14e001acb78f68 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 8 Jan 2025 11:42:27 +0100
Subject: [PATCH] arm64: dts: st: correct imx335 reset gpio polarity in
 stm32mp257f-dk.dts

With commit 9eac174a9b89 ("media: imx335: Fix reset-gpio handling")
introduction, reset gpio line must be corrected within the device-tree
in order to match with expected behavior of the reset of module.

Change-Id: I0a757448370c35830261ef0962095dd46538441d
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -420,7 +420,7 @@
 		avdd-supply = <&imx335_2v9>;
 		ovdd-supply = <&imx335_1v8>;
 		dvdd-supply = <&imx335_1v2>;
-		reset-gpios = <&gpiob 1 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL)>;
+		reset-gpios = <&gpiob 1 (GPIO_ACTIVE_LOW | GPIO_PUSH_PULL)>;
 		powerdown-gpios = <&gpiob 11 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL)>;
 		status = "okay";
 
